plugins/by-name/wrapping/default.nix

```
@@ -0,0 +1,26 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "wrapping";
+  originalName = "wrapping.nvim";
+  package = pkgs.vimPlugins.wrapping-nvim;
+
+  maintainers = [ lib.maintainers.ZainKergaye ];
+
+  # Optionally, explicitly declare some options. You don't have to.
+  settingsOptions.notify_on_switch = defaultNullOpts.mkBool true ''
+       By default, wrapping.nvim will output a message to the
+    command line when the hard or soft mode is set.
+  '';
```

```suggestion
  settingsOptions = {
    notify_on_switch = defaultNullOpts.mkBool true ''
       By default, wrapping.nvim will output a message to the
       command line when the hard or soft mode is set.
    '';
  };
```
Even if there is only a single option for now, this leaves us the room to add more later (purely cosmetic).